src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleBedDefender.kt

```diff
@@ -92,7 +92,7 @@ object ModuleBedDefender : Module("BedDefender", category = Category.WORLD) {
         }
 
         best
-    }, false)
+    }, false).tree()
```

Comment:
To be honest, that makes no sense either. Now we think we call tree() on Instance of [BlockPlacer], which automatically magic trees himself on to [ModuleBedDefender], since it is not clear that there is an Extension on it.